### ISSUE TITLE
Game renders too large with 4K screen

### ISSUE THREAD

**Issue by AmonBune** [teeworlds/teeworlds#1505]
When launching Teeworlds on a Computer with a resolution of 3840x2160 the game renders too large and the mouse always stays in the top left corner of the screen.

Screenshot of the issue (Yes, it's a screenshot of the whole monitor!):
![image](https://user-images.githubusercontent.com/15927695/32137723-cba60022-bc25-11e7-8ba9-2a460772e21a.png)

settings.cfg:
```
cl_predict 1
cl_nameplates 1
cl_nameplates_always 1
cl_nameplates_teamcolors 1
cl_nameplates_size 50
cl_autoswitch_weapons 0
cl_showhud 1
cl_show_chat_friends 0
cl_showfps 0
cl_airjumpindicator 1
cl_threadsoundloading 0
cl_warning_teambalance 1
cl_mouse_deadzone 300
cl_mouse_followfactor 60
cl_mouse_max_distance 800
ed_showkeys 0
cl_show_welcome 0
cl_motd_time 10
cl_version_server "version.teeworlds.com"
cl_languagefile ""
player_use_custom_color 0
player_color_body 65408
player_color_feet 65408
player_skin "default"
ui_page 6
ui_toolbox_page 0
ui_server_address "localhost:8303"
ui_scale 100
ui_mousesens 100
ui_color_hue 160
ui_color_sat 70
ui_color_lht 175
ui_color_alpha 228
gfx_noclip 0
player_name "nameless tee"
player_clan ""
player_country -1
logfile ""
cl_cpu_throttle 0
cl_load_country_flags 1
cl_auto_demo_record 0
cl_auto_demo_max 10
cl_auto_screenshot 0
cl_auto_screenshot_max 10
inp_grab 0
br_filter_string ""
br_filter_full 0
br_filter_empty 0
br_filter_spectators 0
br_filter_friends 0
br_filter_country 0
br_filter_country_index -1
br_filter_pw 0
br_filter_ping 999
br_filter_gametype ""
br_filter_gametype_strict 0
br_filter_serveraddress ""
br_filter_pure 1
br_filter_pure_map 1
br_filter_compatversion 1
br_sort 4
br_sort_order 1
br_max_requests 25
snd_buffer_size 512
snd_rate 48000
snd_enable 1
snd_enable_music 1
snd_volume 100
snd_device -1
snd_nonactive_mute 0
gfx_screen_width 3840
gfx_screen_height 2160
gfx_borderless 0
gfx_fullscreen 1
gfx_alphabits 0
gfx_color_depth 24
gfx_clear 0
gfx_vsync 1
gfx_display_all_modes 0
gfx_texture_compression 0
gfx_high_detail 1
gfx_texture_quality 1
gfx_fsaa_samples 0
gfx_refresh_rate 0
gfx_finish 1
gfx_asyncrender 0
gfx_threaded 0
inp_mousesens 100
unbindall
bind tab "+scoreboard"
bind space "+jump"
bind 1 "+weapon1"
bind 2 "+weapon2"
bind 3 "+weapon3"
bind 4 "+weapon4"
bind 5 "+weapon5"
bind a "+left"
bind d "+right"
bind t "chat all"
bind u "+show_chat"
bind y "chat team"
bind right "spectate_next"
bind left "spectate_previous"
bind f1 "toggle_local_console"
bind f2 "toggle_remote_console"
bind f3 "vote yes"
bind f4 "vote no"
bind f10 "screenshot"
bind rshift "+spectate"
bind lshift "+emote"
bind mouse1 "+fire"
bind mouse2 "+hook"
bind mousewheelup "+prevweapon"
bind mousewheeldown "+nextweapon"
```

**Comment by IBPX** [comment]
Could you post your settings.cfg file? It should be in ~/.teeworlds/ on Linux.

**Comment by pielover88888** [comment]
Find the executable (teeworlds.exe), `right click -> properties` look for the tab that says Compatibility and change HiDPI scaling.

**Comment by AmonBune** [comment]
@pielover88888 Thanks, that works as a workaround. But it would still be good if the game could handle High DPI displays without manually setting any compatibility mode.

**Comment by oy** [comment]
Fixed by 9b22b2426daf71bac6078a2376544a96cd7f06b4
